src/Shared/Recruit.Vacancies.Client/Application/Commands/ReviewVacancyCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Commands
+{
+    public class ReviewVacancyCommand : ICommand, IRequest
+    {
+        public ReviewVacancyCommand(Guid vacancyId, VacancyUser user, OwnerType submissionOwner)
+        {
+            VacancyId = vacancyId;
+            User = user;
+            SubmissionOwner = submissionOwner;
+        }
+
+        public ReviewVacancyCommand(Guid vacancyId, VacancyUser user, string employerDescription, OwnerType submissionOwner)
```

Comment:
employerDescription could be an optional param as opposed to 2 constructors?

Reply:
Done